src/Microsoft.AspNet.Razor/Parser/SyntaxTree/Block.cs

```diff
@@ -19,7 +19,7 @@ public Block(BlockBuilder source)
             source.Reset();
         }
 
-        protected Block(BlockType? type, IEnumerable<SyntaxTreeNode> contents, IParentChunkGenerator generator)
+        protected Block(BlockType? type, List<SyntaxTreeNode> contents, IParentChunkGenerator generator)
```

Comment:
Oh shoot, this is `Block`. Yes `IReadOnlyList` works. Ignore my previous comment.